SAMtags.tex

```diff
@@ -208,6 +212,16 @@ \subsection{Additional Template and Mapping data}
 \subsection{Metadata}
 
 \begin{description}
+
+\item[FT:Z:\tagvalue{text}] Filter field, free-text indicating the reason why this record has been modified or filtered
```

Comment:
I'm thinking that FT is perhaps a bad tag for this. perhaps CO (COmment) would be better as it is akin to the @CO tag in the header, and you can put whatever free text you want there to explain something you did to the read.